MAUI/Masked-Entry/Basic-Features.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Basic Features in .NET MAUI Masked Entry control | Syncfusion
+description:  Learn here all about Basic Features support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# Basic Features in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Setting ClearButtonVisibility
+
+ClearButtonVisibility, of type ClearButtonVisibility, controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000"
+                          ClearButtonVisibility = "Visible" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.ClearButtonVisibility = Visible;
+
+{% endhighlight %}
+{% endtabs %}
+
+![ClearButtonVisibility](MaskedEntry_Images/MaskedEntry_ClearButtonVisibility.png)
+
+## Setting CursorPosition
+
+Cursor position in the masked entry can either be obtained or updated using `CursorPosition` property in SfMaskedEntry.
```

Comment:
The cursor position in the masked entry can either be obtained or updated using the `CursorPosition` property in SfMaskedEntry.

Reply:
Addressed the changes.